src/main/java/twofive/exception/EmptyDateException.java

```diff
@@ -0,0 +1,12 @@
+package twofive.exception;
+
+/**
+ * Represents an error encountered when a date is not provided for parsing
+ * in a DueCommand.
+ */
```

Comment:
Extensive handling of exceptions, which is hallmark of good code but consider adding an overarching DukeException class to pass exceptions to?